Frontend/android-native/app/src/driver/nammaYatriPartner/assets/juspay/v1-configuration.js

```diff
@@ -262,6 +262,28 @@ window.getMerchantConfig = function () {
 
           "auto" : {},
           "bike" : {}
+        },
+        "travelMode" : {
+          "pickup" : {
+            "autoRickshaw" : "DRIVE",
+            "taxi" : "DRIVE",
+            "taxiPlus" : "DRIVE",
+            "sedan" : "DRIVE",
+            "suv" : "DRIVE",
+            "hatchback" : "DRIVE",
+            "bike" : "DRIVE",
+            "suvPlus" : "DRIVE"
+          },
+          "dropOff" : {
+            "autoRickshaw" : "DRIVE",
+            "taxi" : "DRIVE",
+            "taxiPlus" : "DRIVE",
+            "sedan" : "DRIVE",
+            "suv" : "DRIVE",
+            "hatchback" : "DRIVE",
+            "bike" : "DRIVE",
+            "suvPlus" : "DRIVE"
+          }
```

Comment:
Add this in other merchants' V1-config also and verify app is opening without decoding fail.

Reply:
You can store this values in variable and use that in all places. you can refer `rideStartAudio`